### ISSUE TITLE
[Feature request] Simple wallet option

### ISSUE THREAD

**Issue by Opago-Pay** [lnbits/lnbits#2785]
It would be great to have to option to hide the api keys and menu bar for non admin users on mobile.

This way the Export to Phone with QR Code could be used to give limited access to users, where they cannot edit any settings nor create new wallets.

**Comment by Opago-Pay** [comment]
Yes, obviously. The point is more to make a shareable browser-based wallet, without giving ordinary users full access, not necessarily to lock out tech savy users.

Thinking about it, maybe it would make sense to not only make this mobile centered, but rather to give admins an option to set simple mode.

Essentially, in simple mode, users cannot create wallets and cannot see anything about extensions unless the admin has activated the extension for the user already.

Basically, the user in simple mode should only see this:
<img width="1297" alt="image" src="https://github.com/user-attachments/assets/810ac463-d3c3-4289-97f0-6ca625603f02">

**Comment by talvasconcelos** [comment]
There's an option to disable the API stuff on the server settings, extensions tab:

![image](https://github.com/user-attachments/assets/777a4f7d-57b4-428d-90a1-510614a61d4d)
